### PR TITLE
feat: implement search function for Faq

### PR DESCRIPTION
**Pull request by favourachara07**
## What I was told to do
Implement the search functionality for the FAQ component using keywords
## What I did
I made changes to the root file(index.tsx)
I added the assets folder that contained the svg and logo
The component and search functionality 
### Screenshots
Screenshot (43)
Screenshot (43)

### CODE REVIEW

**Comment by chumex412** [comment]
Resolve the conflicts

File: app/routes/_index.tsx [hunk 1]
```diff
@@ -1,50 +1,671 @@
...
+              <img className="w-8 h-8" src={hng} alt="" />
+              <span className="self-center text-2xl font-semibold whitespace-nowrap dark:text-white">
+                HNG Boilerplate
+              </span>
+            </Link>
+            <div id="mobile-nav" className="flex md:order-2">
+              <div className="relative hidden md:block">
+                <button
+                  type="button"
+                  className="text-black hover:text-white border border-orange-600 hover:bg-orange-600 focus:ring-4 focus:outline-none focus:ring-orange-300 font-medium rounded-md text-sm px-5 py-2.5 text-center me-2 mb-2 "
```
Comment (chumex412) [inline app/routes/_index.tsx:92]: Why are you implementing your own nav bar?

File: app/routes/_index.tsx [hunk 2]
```diff
@@ -1,50 +1,671 @@
...
+                <li>
+                  {/* dark:text-white md:dark:hover:text-blue-500 dark:hover:bg-gray-700 dark:hover:text-white md:dark:hover:bg-transparent dark:border-gray-700 */}
+                </li>
+              </ul>
+            </div>
+          </div>
+        </nav>
+      </header>
+
+      <main>
```
Comment (chumex412) [inline app/routes/_index.tsx:173]: is this the proper route for the FAQ? Please use the proper route.

File: app/routes/_index.tsx [hunk 3]
```diff
@@ -1,50 +1,671 @@
...
+      </form>
+
+      <footer className="bg-gray-100 dark:bg-gray-900 mt-10">
+        <div className="mx-auto w-full max-w-screen-xl p-4 py-6 lg:py-8">
+          <div className="md:flex md:justify-evenly">
+            <div className="mb-6 md:mb-0 mr-10">
+              <Link to="/" className="flex items-center justify-center">
+                <div className="flex flex-col">
+                  <span className="self-center text-2xl font-semibold whitespace-nowrap dark:text-white mb-5">
+                    BoilerPlate
```
Comment (chumex412) [inline app/routes/_index.tsx:321]: Why are you creating the footer yourself, please look for the footer and nav bar components and use those instead

**Review by chumex412 — CHANGES_REQUESTED** [review]

**Comment by favourachara07** [comment]
> Resolve the conflicts

I have resolved them
Please your review

**Review by shalomtaiwo — DISMISSED** [review]
Please resolve conflict

**Review by SirhmVFX — APPROVED** [review]

**Comment by SirhmVFX** [comment]
your branch has conflicts... resolve them

**Review by chumex412 — APPROVED** [review]

**Review by chumex412 — APPROVED** [review]

**Review by chumex412 — APPROVED** [review]

**Pull request merged**